torch/export/_trace.py

```diff
@@ -1366,6 +1563,281 @@ def forward(self, *args, **kwargs):
     )
 
 
+@_log_export_wrapper
+@_disable_prexisiting_fake_mode
+def _export_for_training(
+    mod: torch.nn.Module,
+    args: Tuple[Any, ...],
+    kwargs: Optional[Dict[str, Any]] = None,
+    dynamic_shapes: Optional[Union[Dict[str, Any], Tuple[Any], List[Any]]] = None,
+    *,
```

Comment:
@tugsbayasgalan could you leave a TODO for tensor conversion .to() might not be preserved during export.